DataStructures/DynamicStacks/UnitTestDynamicStack.java

```diff
@@ -0,0 +1,76 @@
+public class UnitTestDynamicStack{
+    public static void main(String[] args){
+        DynamicStack ds = new DynamicStack();
+        DynamicStack dswm = new DynamicStack(5);
+
+
+
+        
+
+        
+        System.out.println("Empty-" + dswm.isEmpty() + " | Current Size-" + dswm.size() + " | Value Added-" + dswm.push(1) + " | Peek-" + dswm.peek() + " | Full-" + dswm.isFull() + " | Node Count-" + dswm.size());
```

Comment:
You could have done this on a loop :)

Reply:
I have made this into a loop.  It's so much more clean now.